### DISCUSSION TITLE
Is English the only supported language in .csv file?

### DISCUSSION THREAD

**Discussion by ssangho8** [netbox-community/netbox-docker#755] [Getting Help]
Hi, 
I installed netbox on my docker system.

When I try to import .csv file which contains Korean text, an error occurs.
![캡처](https://user-images.githubusercontent.com/49374006/166189780-41366c5e-1d6d-4f5a-9a22-5abae7dad324.PNG)

If I add an contents one by one, Korean works fine.

Is there any settings that I have to change to use Korean language?

**Comment by kkthxbye-code** [comment]
The contents of the csv file has to be utf-8. If you need help, please provide an failing example csv file.

**Reply by ssangho8** [reply]
Here's an example

[utf_Import_test.csv](https://github.com/netbox-community/netbox-docker/files/8601704/utf_Import_test.csv)

**Reply by kkthxbye-code** [reply]
That's not UTF-8, looks like cp949 or similar. Export the csv file as utf-8 and try again.

**Reply by ssangho8** [reply]
I saved it as utf-8 for sure

**Reply by ssangho8** [reply]
Problem solved. 

I put data in notepad and saved it as UTF-8.

File name needs to be saved as ~.csv . 

Thanks for help.
